net-mgmt/nrpe3/src/opnsense/mvc/app/controllers/OPNsense/Nrpe/forms/general.xml

```diff
@@ -0,0 +1,54 @@
+<form>
+    <field>
+        <id>general.enabled</id>
+        <label>Enable NRPE</label>
+        <type>checkbox</type>
+        <help>This will activate the NRPE service.</help>
+    </field>
+    <field>
+        <id>general.server_port</id>
+        <label>Server Port</label>
+        <type>text</type>
+        <help>Set the port the daemon is listening to.</help>
+    </field>
+    <field>
+        <id>general.server_address</id>
+        <label>Server Address</label>
+        <style>tokenize</style>
+        <type>select_multiple</type>
+        <allownew>true</allownew>
+        <help>Set the address the daemon is listening to. Empty means listen to all addresses.</help>
+    </field>
+    <field>
+        <id>general.allowed_hosts</id>
+        <label>Allowed Hosts</label>
+        <style>tokenize</style>
+        <type>select_multiple</type>
+        <allownew>true</allownew>
+        <help>Set the list of allowed hosts which can query this system.</help>
+    </field>
+    <field>
+        <id>general.dont_blame_nrpe</id>
+        <label>Allow Arguments</label>
+        <type>checkbox</type>
+        <help>Enable to allow arguments sent to NRPE.</help>
```

Comment:
From the name I guess this action is dangerous. Should be mentioned in help string.